templates/tg-mini-app-seamless-signing-boilerplate-template/frontend/components/WalletSelector.tsx

```diff
@@ -0,0 +1,91 @@
+import { Mizu } from "@mizuwallet-sdk/core";
+import { truncateAddress } from "@aptos-labs/wallet-adapter-react";
+import { Network } from "@aptos-labs/ts-sdk";
+import { Copy, LogOut } from "lucide-react";
+import { useCallback, useEffect } from "react";
+// Internal components
+import { Button } from "@/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { useToast } from "@/components/ui/use-toast";
+import { useMizuWallet } from "@/components/WalletProvider";
+import { MIZU_WALLET_APP_ID } from "@/constants";
+
+export function WalletSelector() {
+  const { mizuClient, setMizuClient, userAddress, setUserAddress } = useMizuWallet();
+
+  const { toast } = useToast();
+
+  const copyAddress = useCallback(async () => {
+    if (!userAddress) return;
+    try {
+      await navigator.clipboard.writeText(userAddress);
+      toast({
+        title: "Success",
+        description: "Copied wallet address to clipboard.",
+      });
+    } catch {
+      toast({
+        variant: "destructive",
+        title: "Error",
+        description: "Failed to copy wallet address.",
+      });
+    }
+  }, [userAddress, toast]);
+
+  useEffect(() => {
+    if (!mizuClient) return;
+    mizuClient.getUserWalletAddress().then((addr) => setUserAddress(addr));
+  }, [mizuClient]);
+
+  const onConnectWallet = async () => {
+    localStorage.setItem("auto-connect", "true");
```

Comment:
this should be set explicitly by the user, right?

Reply:
when user click connect for the first time, this will be set.
by default `auto-connect` is undefine